### ISSUE TITLE
2014 Primary Merced test failure

### ISSUE THREAD

**Issue by charles-difazio** [openelections/openelections-data-ca#107]
Missing write-in candidate. Can't find any docs from the county that contains precinct-level or county-level write-in results.

```
E           AssertionError: Merced failed
E           assert {('"Bo" Bogda...r'): 390, ...} == {('"Bo" Bogdan...r'): 390, ...}
E             Omitting 68 identical items, use -v to show
E             Left contains more items:
E             {('Jack Mobley', 'Merced', 21.0, 'State Assembly'): 1220}
E             Use -v to get the full diff
```

**Comment by carbonphyber** [comment]
Fixed in #161

**Comment by carbonphyber** [comment]
Fixed in #175

**Comment by warwickmm** [comment]
I believe this can be closed.
